features/login.feature

```diff
@@ -9,10 +9,10 @@ Feature: Log in as an admin, as a teacher, as a student
         As an admin, I want to be able to login
         When admin logs in with email <adminUsername> and password <password>
         Then admin accepts data protection
-        Then user-dashboard should have the correct initials
-        Then user-dashboard should have the correct school
-        Then user-dashboard should have the correct name and profession
```

Comment:
why is not yet correctly implemented, because in this step we check full name and role (profession) of user

Reply:
the method behind "user-dashboard should have the correct name and profession" did the same as "user-dashboard should have the correct initials"